web/src/pages/apd/ApdReadOnly.js

```diff
@@ -17,7 +17,7 @@ import ApdSummary from '../../containers/viewOnly/ApdSummary';
 import PreviousActivities from './previous-activities/PreviousActivities';
 import Activities from '../../containers/viewOnly/activities/All';
 import ScheduleSummary from './schedule-summary/ScheduleSummaryReadOnly';
-import ProposedBudget from '../../containers/viewOnly/ProposedBudget';
+import ProposedBudget from './proposed-budget/ProposedBudget';
```

Comment:
I think you are using the wrong one here, should be ProposedBudgetReadOnly

Reply:
ahhh!!! keep making the same mistake. will make sure to double check this once I create the PR :)

Reply:
just pushed up the fix